packages/clerk-js/src/core/clerk.ts

```diff
@@ -585,7 +587,7 @@ export default class Clerk implements ClerkInterface {
     return await customNavigate(stripOrigin(toURL));
   };
 
-  public buildUrlWithAuth(to: string): string {
+  public buildUrlWithAuth(to: string, strict = true): string {
```

Comment:
@panteliselef Can you please elaborate why do we need the `strict` parameter in this function?

Reply:
@SokratisVidros  This was an attempt to prevent the error from throwing for the following scenario.

- User lands on the satellite website (first time visit)
- Redirecting to primary website (first time visit)
- Primary does not have a _dev_session in localstorage
- redirectWithAuth for satellite will throw an error because _dev_session was not found.